tests/test_wave_01.py

```diff
@@ -158,13 +194,14 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
-    
-    raise Exception("Test needs to be completed.")
+    assert updated_data.get("watched")[0]["title"] == MOVIE_TITLE_1 
```

Comment:
Great start! It would be good to check that the other items in the dictionary are what you expect as well!